tests/phpunit/tests/block-template-utils.php

```diff
@@ -24,7 +24,7 @@ public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
 				'post_type'    => 'wp_template',
 				'post_name'    => 'my_template',
 				'post_title'   => 'My Template',
-				'post_content' => 'Content',
+				'post_content' => '<!-- wp:tests/anchor-block -->Hello<!-- /wp:tests/anchor-block -->',
```

Comment:
I'd like to keep these distinct from the [updated value that "comes in from the wire"](https://github.com/WordPress/wordpress-develop/pull/6254/files#diff-d5470ce0db8dafe9e9d34bc2dd1e9c5d3f67bdfd365427b91a0ab438b0382314R494) in the new unit test. Otherwise, the test loses some value, as we won't be able to tell if the correct template object was given to the filter (i.e. one that's based on the `$changes` or the one that was already present in the DB).

If we really need a block here to make sure it's traversed, we can maybe add a different one; although I'd hope we can maybe just revert to `Content` and instead change the [test](https://github.com/WordPress/wordpress-develop/pull/6254/files#diff-d5470ce0db8dafe9e9d34bc2dd1e9c5d3f67bdfd365427b91a0ab438b0382314R500-R502) (so it uses `end( $args )` to look at the arguments from the last function call, rather than `$args[0]`).

Reply:
>instead change the [test](https://github.com/WordPress/wordpress-develop/pull/6254/files#diff-d5470ce0db8dafe9e9d34bc2dd1e9c5d3f67bdfd365427b91a0ab438b0382314R500-R502) (so it uses end( $args ) to look at the arguments from the last function call, rather than $args[0]).

This would work and satisfy the test.

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/6254/commits/d5d8b9a6ea3161e2e1b367568f73731d0a08ce6e